pages/balance-budget.vue

```diff
@@ -267,31 +183,79 @@ import FiscalYearSelect from "@/components/FiscalYearSelect";
 import Header from "@/components/Header";
 import Footer from "@/components/Footer";
 import DepartmentsWalkthrough from "@/components/DepartmentsWalkthrough";
+import { D3PieChart } from 'vue-d3-charts';
 
+var total_expenses = 1234.0
+var starting_expense = total_expenses / 7.0
```

Comment:
nit: even though this is dummy data, `var` is somewhat deprecated. because these values won't change, it's better to use `const`
```suggestion
const total_expenses = 1234.0
const starting_expense = total_expenses / 7.0
```